src/bio.c

```diff
@@ -1755,7 +1755,7 @@ int wolfSSL_BIO_reset(WOLFSSL_BIO *bio)
  * @param close_flag BIO_NOCLOSE or BIO_CLOSE
  * @return New BIO object or NULL on failure
  */
-WOLFSSL_BIO *wolfSSL_BIO_new_fd(int fd, int close_flag)
+WOLFSSL_BIO *wolfSSL_BIO_new_fd(SOCKET_T fd, int close_flag)
```

Comment:
OpenSSL API takes `int`. Leave it as is and cast inside.

Reply:
:heavy_check_mark: